radicle/src/storage/refs.rs

```diff
@@ -101,6 +101,17 @@ impl Refs {
         })
     }
 
+    /// Get a particular ref.
+    pub fn get(&self, name: &git::Qualified) -> Option<Oid> {
+        self.0.get(name.to_ref_string().as_refstr()).copied()
+    }
+
+    /// Get a particular head ref.
+    pub fn head(&self, name: &git::Component) -> Option<Oid> {
```

Comment:
I think you'd want `RefString` or `AsRef<RefStr>`, otherwise you can't do something like `finto/feature/patches`

Reply:
Oh shit you're right!

Reply:
Fixed.